### PR TITLE
Add VaultCord

### PR DESCRIPTION
**Pull request by wnelson03**
VaultCord is a service that allows you to backup & restore channel settings, server settings, and server members on the communication platform Discord.

It also has an extensive API and is used by programmers, which is why I put VaultCord under the developer category also https://docs.vaultcord.com/api-reference/introduction

**Documentation** https://docs.vaultcord.com/misc/multi-factor-authentication

Similarweb: We use multiple domains in different regions, our domain `novanode.win` is ranked 144,982 globally https://www.similarweb.com/website/novanode.win/#overview
On the Similarweb picture, it shows that the domain is used for VaultCord.

Additionally, here is DNS proof of novanode.win being associated with vaultcord.com https://mxtoolbox.com/SuperTool.aspx?action=txt%3a2factorauth.novanode.win&run=toolpage

image

Our website supports security keys which is a rare, though very important security measure. Hopefully we have satisfied the requirements of 2fa.directory to be listed, thank you for your time 🙂

### CODE REVIEW

**Comment by Carlgo11** [comment]
Thank you for the pull request @wnelson03! 
The documentation mentions magic links and passkeys along with 2FA codes.
Does this mean that the magic link and passkeys are used instead of a password? Or are they used along with a password?

If passkeys replace the need to enter a password, your entry would be a better fit for https://github.com/2factorauth/passkeys

**Comment by wnelson03** [comment]
@Carlgo11 Along with a password. The user must enter a valid password first, then they are prompted to complete multi-factor authentication, and then if completed successfully, the user is logged in.

There are no choices to login with only passkey I can confirm that 👍 password is still needed.

File: entries/v/vaultcord.com.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "VaultCord": {
+    "domain": "vaultcord.com",
```
Comment (Carlgo11) [inline entries/v/vaultcord.com.json:3]: The additional-domains element helps link any other domains to the service.
```suggestion
    "domain": "vaultcord.com",
    "additional-domains": [
      "novanode.win"
    ],
```

File: entries/v/vaultcord.com.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "tfa": [
+      "email",
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://docs.vaultcord.com/misc/multi-factor-authentication",
+    "categories": [
+      "backup",
+      "developer"
```
Comment (Carlgo11) [inline entries/v/vaultcord.com.json]: I understand your reasoning for adding the developer category, but many services provide users with an API and most of them are not developer tools. I think Backup & Sync is the proper category for your service.
```suggestion
      "backup"
```

File: img/v/vaultcord.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...4 C 286.480 275.726,285.988 275.801,286.707 275.890 " stroke="none" fill="#959df6" fill-rule="evenodd"></path><path id="path4" d="M295.700 177.076 C 296.085 177.176,296.715 177.176,297.100 177.076 C 297.485 176.975,297.170 176.893,296.400 176.893 C 295.630 176.893,295.315 176.975,295.700 177.076 M303.300 177.076 C 303.685 177.176,304.315 177.176,304.700 177.076 C 305.085 176.975,304.770 176.893,304.000 176.893 C 303.230 176.893,302.915 176.975,303.300 177.076 M204.109 208.000 C 204.109 208.990,204.187 209.395,204.282 208.900 C 204.378 208.405,204.378 207.595,204.282 207.100 C 204.187 206.605,204.109 207.010,204.109 208.000 M299.300 237.076 C 299.685 237.176,300.315 237.176,300.700 237.076 C 301.085 236.975,300.770 236.893,300.000 236.893 C 299.230 236.893,298.915 236.975,299.300 237.076 M282.307 275.490 C 283.026 275.579,284.106 275.577,284.707 275.486 C 285.308 275.395,284.720 275.322,283.400 275.324 C 282.080 275.326,281.588 275.401,282.307 275.490 M290.705 275.483 C 291.202 275.579,291.922 275.575,292.305 275.475 C 292.687 275.375,292.280 275.296,291.400 275.301 C 290.520 275.305,290.207 275.387,290.705 275.483 " stroke="none" fill="#acb4fc" fill-rule="evenodd"></path></g></svg>
```
Comment (Carlgo11) [inline img/v/vaultcord.com.svg]: It looks like you've run a PNG through some sort of vectorization tool. Is that correct?
Vector images can be turned into rasterized images, but the opposite can't be done in a good way.
If you have the original vector graphics somewhere, try converting it to SVG. If not, replace this image with a PNG image that's 32x32, 64x64 or 128x128 pixels and compress it with [TinyPNG](https://tinypng.com/) before submitting.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
I don't think the `vc` is vertically centered in the image, but if you're happy with how it looks, I'll go ahead and merge. Let me know if you're happy with the PR as it is right now.

**Comment by wnelson03** [comment]
@Carlgo11 Thank you for your help, yes I'm happy with how it looks. Please merge, I appreciate it 👍

**Pull request merged**